### PR TITLE
chore: specify node version that doesn't break examples.spec.jsx

### PR DESCRIPTION
**Pull request by vdnj**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,9 @@
     "url": "https://github.com/cozy/cozy-ui/issues"
   },
   "homepage": "https://github.com/cozy/cozy-ui",
+  "engines": {
+    "node": ">=16.16"
+  },
```
Comment (Crash--) [inline package.json:21]: I'm not a big fan of requiring node 16.16 for cozy-ui. Can you tell us the issue you had with older node version please?

Comment (Crash--) [reply package.json:21]: By settings engines node >= 16.16 you tell that ALL stuff that consume that cozy-ui should at least be on this version. So you can't install cozy-ui in your project if you still use node 12/14/16.14 even if there is no need of 16.16 for cozy-ui for this application. It a bit brutal. 

We only need >=16.16 for our dev process, right?

**Comment by JF-Cozy** [comment]
@vdnj petit up pour cette PR ?

File: package.json (hunk 1 above)
Comment (vdnj) [reply package.json:21]: @trollepierre do you remember why older version breaks examples.spec.js ? Is it a better way to solve it ?

Comment (JF-Cozy) [reply package.json:21]: si c'est pour du dev, seul la modif dans nvmrc suffit non ? Faudrait peut-être rajouter aussi dans travis 🤔

Comment (trollepierre) [reply package.json:21]: @JF-Cozy : https://github.com/cozy/cozy-ui/blob/master/.nvmrc the .nvmrc is updated.

Comment (trollepierre) [reply package.json:21]: > do you remember why older version breaks examples.spec.js ? Is there a better way to solve it ?

No. It was an async error, that we usually faced when Node version is not correct. 
I asked you to launch: `nvm use 16.16 && yarn test` and that solved the problem. 
But I think you said later that you still encounter the problem again, don't you?

Comment (vdnj) [reply package.json:21]: Yes

Comment (trollepierre) [reply package.json:21]: On this branch, or without this fix?

Comment (trollepierre) [reply package.json:21]: then, let's close this PR.

**Comment by JF-Cozy** [comment]
je close donc

**Pull request closed without merging**